modules/csg/csg_shape.cpp

```diff
@@ -2003,6 +2007,9 @@ void CSGPolygon::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_path_rotation", "mode"), &CSGPolygon::set_path_rotation);
 	ClassDB::bind_method(D_METHOD("get_path_rotation"), &CSGPolygon::get_path_rotation);
 
+	ClassDB::bind_method(D_METHOD("set_path_local", "local"), &CSGPolygon::set_path_local);
+	ClassDB::bind_method(D_METHOD("get_path_local"), &CSGPolygon::get_path_local);
```

Comment:
Boolean getters should start with `is_`. Also the argument for the setter would be better as `enabled` than `local`.

Reply:
so is_path_local and set_path_local?